### PR TITLE
Some minor fixes for #3

### PR DESCRIPTION
**Pull request by jakirkham**
Sorry, I missed this one.

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -64,8 +64,8 @@ if [ ! -n "$WERCKER_EMAIL_NOTIFY_BODY" ]; then
   DEFAULT_BODY="$DEFAULT_BODY\nGit Commit: \"$GIT_COMMIT_MESSAGE\" ($WERCKER_GIT_COMMIT)"
   DEFAULT_BODY="$DEFAULT_BODY\nStarted By: $WERCKER_STARTED_BY"
 
-  export WERCKER_EMAIL_NOTIFY_BODY
   WERCKER_EMAIL_NOTIFY_BODY=$(echo -e "$DEFAULT_BODY")
+  export WERCKER_EMAIL_NOTIFY_BODY
```
Comment (flenter) [inline run.sh]: Why switch this?

Comment (jakirkham) [inline run.sh]: No good reason. Why can't they be on the same line?

**Comment by flenter** [comment]
Another PR :+1: Changes look good to me, apart from one question.

**Comment by jakirkham** [comment]
Ok, tried bring the export back with the assignment. Was this causing a problem previously?

**Comment by flenter** [comment]
It's was done after a suggestion by shellcheck, a tool we use to help detect issues in shell scripts: 
https://github.com/koalaman/shellcheck/wiki/SC2155

**Comment by jakirkham** [comment]
Ah, ok. Thanks for that.

I have reverted that change completely. Should be go to go.

**Comment by jakirkham** [comment]
Should fix shellcheck "missing Shebang (#!)" error. ( https://github.com/koalaman/shellcheck/wiki/SC2148 )

**Pull request merged**